vignettes/articles/cheatsheet.Rmd

```diff
@@ -0,0 +1,557 @@
+---
+title: "Relational data with dm: Cheat sheet"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cheatsheet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+resource_files:
+  - cheatsheet
```

Comment:
this is how the printable cheatsheet and all its files gets added to the pkgdown website.
I found no other way since we can't know where the pkgdown website is built (it's built in a temporary folder).